### DISCUSSION TITLE
OTP input

### DISCUSSION THREAD

**Discussion by AYGTX** [primefaces/community#789] [PrimeNG] [answered]
Hi team, 
Would you please consider adding an OTP input field, which takes alphanumerics or numbers with a custom length 
Reference :
https://www.codehim.com/wp-content/uploads/2023/09/OTP-Input-Field-Using-Bootstrap-5.png

**Comment by cetincakiroglu** [comment, marked as answer]
Hi,

The link is broken. OTP input can be created by using

- inputText or inputnumber component
- card

and custom contents. You can create on your own by using PrimeNG

**Reply by AYGTX** [reply]
Thanks, I see this is added now !
